### PR TITLE
replace gs:// prefixes w/ MS URIs

### PR DESCRIPTION
**Pull request by leondz**
as indicated in readme - also, readme indication now removed

### CODE REVIEW

**Comment by WuTheFWasThat** [comment]
thanks!  

but... are you sure this is everything?  there's another reference in the README, at a minimum.  I think it's best to either keep the original warning or to be 100% sure everything is covered (hopefully `git grep 'gs://'` should turn up nothing!)

kinda sad if this was the only change required haha, sorry I never bothered to test

**Comment by WuTheFWasThat** [comment]
there's also a bunch of functions in lm_human_preferences/utils/gcs.py, not sure if places they're used broke?

**Comment by leondz** [comment]
my oh my. such hardcoding! ;) give it a few minutes + i'll update

**Review by leondz — COMMENTED** [review]
Alright, I'll be frank, I'm mostly here for the data, & less here for un- and re-wrapping gcs :)

At this update:
- gcs is still in the code
- not sure what to do with the gpt-2 uris
- tldr dataset uris FIXED
- the expected links return 404 for book_passages (https://openaipublic.blob.core.windows.net/lm-human-preferences/labels/datasets/book_passages/{mode}.jsonl), what files are in that dir? tried fishing around a bit, not even sure this dir exists. please help @WuTheFWasThat , i don't have access to look in that dir structure for `book_passages/train.json` and `book_passages/valid.json`
- cnndm data uris FIXED, cache uri not fixed (it's part of gcs)

**Comment by WuTheFWasThat** [comment]
i mean, you should feel free to not fix everything, so long as you keep the original notice :)

**Comment by WuTheFWasThat** [comment]
yeah... can't find the books stuff.  probably should just delete.  

i do see `az://openaipublic/lm-human-preferences/datasets/shortstories/shortstories_v0_{split}.jsonl` but not sure that's the same

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 **Status:** Archive (code is provided as-is, no updates expected)
 
-**Status:** All references to `gs://lm-human-preferences/` should be updated to `https://openaipublic.blob.core.windows.net/lm-human-preferences/labels`.  The code provided as is no longer works.  Pull requests welcome
```
Comment (WuTheFWasThat) [inline README.md:3]: probably should be something like
```
**Status:** All references to Google Cloud Storage have been replaced with Azure. 
 As a result, some functionality may no longer work. Pull requests welcome
 ```

**Comment by leondz** [comment]
> i mean, you should feel free to not fix everything, so long as you keep the original notice :)

yeah, that's valid, restored!

File: lm_human_preferences/train_policy.py [hunk 2]
```diff
@@ -426,7 +426,7 @@ def train(hparams: HParams):
         hyperparams.dump(m.hparams(), name='model_hparams')
 
         if save_dir:
-            if not save_dir.startswith('gs://'):
```
Comment (WuTheFWasThat) [inline lm_human_preferences/train_policy.py:429]: no idea if tf.gfile.Open works with azure URIs so probably revert this part?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+Status: All references to gs://lm-human-preferences/ should be updated to https://openaipublic.blob.core.windows.net/lm-human-preferences/labels. The code provided as is no longer works. Pull requests welcome
```
Comment (WuTheFWasThat) [inline README.md:3]: oof the original notice also had a silly bug of having `/labels` in the azure, when really that only applies if the GCS link had `../labels/...`

Comment (leondz) [reply README.md:3]: this was still sometimes true. did the best fishing i could

**Pull request merged**